Caddyfile

```diff
@@ -37,7 +37,7 @@
         path_regexp /preview/chrome(.*)
     }
     handle @preview_match {
-        uri strip_prefix /beta/chrome
+        uri strip_prefix /preview/chrome
```

Comment:
Is it possible to keep both? There will be a period where both /beta and /preview should be active.